Resources/Storage/StorageElement.py

```diff
@@ -166,8 +166,8 @@ def __init__( self, name, plugins = None, vo = None ):
     self.log = gLogger.getSubLogger( "SE[%s]" % self.name )
     self.useCatalogURL = gConfig.getValue( '/Resources/StorageElements/%s/UseCatalogURL' % self.name, False )
 
+    #                         'getTransportURL',
     self.readMethods = [ 'getFile',
-                         'getTransportURL',
```

Comment:
These are the only two changes in this file: remove getTransportURL from readMethods and ad it to okMethods